lib/creek/styles/value_convert/custom_number_convert.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Creek
+  class Styles
+    module ValueConvert
+      class CustomNumberConvert < Base
+        def initialize(columns: [])
+          @columns = columns
+        end
+
+        def target_cell?(cell_name)
+          cell_column_name = cell_name[/[A-Z]+/]
```

Comment:
Is this the expected behavior?
```ruby
'are you SURE about that?'[/[A-Z]+/] #=> "SURE"
```

Reply:
The value here can only be like "ABC123"

Reply:
Maybe it should be moved to utility class or module?